### ISSUE TITLE
Service restart/reload on link change

### ISSUE THREAD

**Issue by ciroiriarte** [linsomniac/drbdlinks#11]
Hello!, 

I recall drbdlinks supporting reload of some services on link creation/delete. Is it possible, to extend that to any arbitrary service?.

Use case would include:
- NginX
- Logrotate
- Rsyslog

Those services won't be managed by a clusterware solution, will always run and need to change behaviour (configuration appears, configuration dissapears) after link modification.

**Comment by linsomniac** [comment]
Yes, that seems fairly possible, and could either be done by allowing for a list of commands to run on link changes, or just supporting a "post" script that is run after links change, and the restart logic could be in there.  Or even something like an "/etc/drbdlinks/post.d" and it would run any scripts found in there.

This is not something I'm currently able to work on I'm afraid.
